partner_fax/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2018 Apruzzese Francesco <f.apruzzese@apuliasoftware.it>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Partner fax',
+    'category': 'Extra Tools',
+    'summary': 'Add fax number on partner',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author':  'Odoo Community Association (OCA)',
+    'website': 'https://odoo-community.org/',
```

Comment:
please replace it by one of these: https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11

Reply:
Done